.circleci/config.yml

```diff
@@ -123,19 +123,37 @@ jobs:
           name: Link NPM sodium
           command: npm link sodium
       - run:
```

Comment:
the order here is incorrect you cannot link sodium before it is installed. move line 122 to 124 below the install argon command

Reply:
done